vhdbuilder/packer/components.json

```diff
@@ -11,6 +11,9 @@
       "downloadURL": "mcr.microsoft.com/oss/kubernetes/dashboard:*",
       "amd64OnlyVersions": [
         "v2.0.0",
+        "v2.0.0",
```

Comment:
why multiple entries

Reply:
it appears to have issue handling versions starting with v
will open work item to track its fix